homeassistant/components/lektrico/strings.json

```diff
@@ -22,6 +22,38 @@
     }
   },
   "entity": {
+    "binary_sensor": {
+      "state_e_activated": {
+        "name": "State e activated"
+      },
+      "overtemp": {
+        "name": "Overtemp"
+      },
+      "critical_temp": {
+        "name": "Critical temp"
+      },
+      "overcurrent": {
+        "name": "Overcurrent"
+      },
+      "meter_fault": {
+        "name": "Meter fault"
+      },
+      "undervoltage": {
+        "name": "Undervoltage"
+      },
+      "overvoltage": {
+        "name": "Overvoltage"
+      },
+      "rcd_error": {
+        "name": "Rcd error"
+      },
+      "cp_diode_failure": {
+        "name": "Cp diode failure"
```

Comment:
whats cp

Reply:
"Cp diode failure" is now called "Ev diode short".